programs/axelar-solana-its/tests/module/deploy_interchain_token.rs

```diff
@@ -709,3 +715,56 @@ async fn test_approve_revoke_approve_deploy_sequence(
 
     Ok(())
 }
+
+#[test_context(ItsTestContext)]
+#[tokio::test]
+async fn test_deploy_interchain_token_authority_with_data_works(
```

Comment:
This test checks the code path for [init_pda_raw_bytes](https://github.com/eigerco/axelar-amplifier-solana/pull/init_pda_raw_bytes)